src/main/resources/db/migration/V1_51__physical_attributes_migration.sql

```diff
@@ -0,0 +1,12 @@
+drop table prison_person_migration_mapping;
+
+create table prison_person_migration_mapping (
+  nomis_prisoner_number          varchar(7),
+  migration_type                 varchar(20),
+  label                          varchar(20)  not null,
+  when_created                   timestamp with time zone not null default now()
+);
+
+create unique index prison_person_migration_pk_index on prison_person_migration_mapping (nomis_prisoner_number, migration_type);
```

Comment:
We need a colmposite primary key of `nomis_prisoner_number` and `migration_type` but that's not support3d by R2DBC. So we're just going with a unique index for now and avoiding any id based processing when using Spring Data. We'll have to see how this goes, but as our use case is simple (write and read back) I'm hoping it wil be fine.

Reply:
Shouldnt the nomis_prisoner_number and migration_type be not null too?

Reply:
I read that by creating a unique index they are implicitly not null - I'll double check that, thanks.